websites.py

```diff
@@ -0,0 +1,10 @@
+patterns = {
+	'pinterest': r'^[a-zA-Z0-9_]{3,15}$',
+	'gitlab': r'^[a-zA-Z0-9_\-.]+$',
+	'github': r'^[a-zA-Z0-9-]{1,39}$',
+	'tumblr': r'^[a-zA-Z0-9-]{1,32}$',
+	'behance': r'^[a-zA-Z0-9_\-]{1,30}$',
+	'instagram': r'^[a-zA-Z0-9_.]+$',
```

Comment:
there will be a limit.  Something will break somewhere.  If you cant find any sane limit, use 999 and include a link in a comment showing that the max length isnt known